app/src/components/account/UserMenu.vue

```diff
@@ -196,6 +283,47 @@ export default {
     getAddressNetwork(address) {
       return this.networks.find((network) => network.id === address.networkId)
     },
+    getSessionName(session) {
```

Comment:
this feels like enriching which should go in keystore js or session js.

Reply:
this is not really enriching here. It is a just a hack to display something when session is neither `local` nor `extension` (and therefore there is no name there at all)

Reply:
OK, just moved this one to the session module